cllc-public-app/Controllers/FileController.cs

```diff
@@ -65,35 +65,70 @@ public FileController(IConfiguration configuration, IHttpContextAccessor httpCon
         /// <param name="entityId"></param>
         /// <param name="isDelete">Some access rules are different for deletes</param>
         /// <returns></returns>
-        private async Task<bool> CanAccessEntity(string entityName, string entityId, bool isDelete = false)
+        private async Task<bool> CanAccessEntity(string entityName, string entityId, string relativeUrl, bool isDelete = false)
         {
             var result = false;
             var id = Guid.Parse(entityId);
+            string folderName = null;
             switch (entityName.ToLower())
             {
                 case "account":
                     var account = await _dynamicsClient.GetAccountByIdAsync(id).ConfigureAwait(true);
-                    result = account != null && CurrentUserHasAccessToAccount(account.Accountid);
+                    if (account != null)
+                    {
+                        result = account != null && CurrentUserHasAccessToAccount(account.Accountid);
```

Comment:
minor nit: `account != null` is redundant here as it was already checked for null above in the `if` statement